tests/cli/test_utils.py

```diff
@@ -327,6 +333,237 @@ def test_validate_assistant_id_in_config(config_file: Text) -> None:
     rasa.shared.utils.io.write_yaml(copy_config_data, config_file, True)
 
 
+def test_data_validate_stories_with_max_history_zero():
+    importer = TrainingDataImporter.load_from_config(
+        "data/test_config/config_defaults.yml",
+        "data/test_moodbot/domain.yml",
+        "data/test_moodbot/data",
+    )
+
+    with pytest.raises(argparse.ArgumentTypeError):
+        rasa.cli.utils.validate_files(False, 0, importer)
```

Comment:
Could we please use the argument names in the function call to improve readability? 🙏🏻